financialaid/models_test.py

```diff
@@ -22,3 +24,20 @@ def test_timestamped_model_update(self):  # pylint: disable=no-self-use
         Tier.objects.filter(id=tier.id).update(name="new_tier")
         third_timestamp = Tier.objects.get(id=tier.id)  # Since we need to re-fetch the object
         assert second_timestamp != third_timestamp
+
+    def test_financial_aid_model_unique(self):
+        """
+        Tests that FinancialAid objects are unique per User and Program
+        """
+        financial_aid = FinancialAidFactory.create()
+        try:
+            # Test creation of FinancialAid that isn't unique_together with "user" and "tier_program__program"
+            FinancialAidFactory.create(user=financial_aid.user)
+            FinancialAidFactory.create(tier_program=financial_aid.tier_program)
+        except ValidationError:
+            self.fail("Creation of FinancialAid objects should have been successful")
```

Comment:
Can you remove the `except` clause here? It would be better to have the original stack trace when the test fails